### DISCUSSION TITLE
Implementing a vanilla buffer for use in firenvim

### DISCUSSION THREAD

**Discussion by MatthewMDavis** [glacambre/firenvim#1632] [Q&A]
I'm excited about the possibility that I can use (n)vim navigation every time I write in my webmail client.

But there's a LOT of stuff that I don't want to show up when I'm working in a textarea. What I really want is just a blank window like the one depicted on the repository's home page. And one that defaults to the markdown filetype, and enters in insert mode.

What's the cleanest way to make it happen? And, is there a way to suppress the loading of all the other plugins (using Lazy)?

**Comment by glacambre** [comment]
It's likely what you need is to make the [selector](https://github.com/glacambre/firenvim?tab=readme-ov-file#configuring-what-elements-firenvim-should-appear-on) used by Firenvim more specific, so that it only takes over the text area of your webmail client instead of something larger that would contain the stuff you don't want.
